app/api/events.py

```diff
@@ -426,7 +426,13 @@ def before_get(self, args, kwargs):
         :param kwargs:
         :return:
         """
+        if 'id' in kwargs:
+            # condition to check if id is actually numerical identifier
+            if len(str(kwargs['id'])) >= 5:
+                new_kwarg = {'identifier': str(kwargs['id'])}
```

Comment:
This will strip all other arguments passed in kwargs

Reply:
@iamareebjamal Okay, that's true, I will fix this up